readthedocs/templates/projects/project_notifications.html

```diff
@@ -2,20 +2,30 @@
 
 {% load i18n %}
 
-{% block title %}{% trans "Edit Notifications" %}{% endblock %}
+{% block title %}{% trans "Email Notifications" %}{% endblock %}
 
 {% block nav-dashboard %} class="active"{% endblock %}
 
 {% block editing-option-edit-proj %}class="active"{% endblock %}
 
 {% block project-notifications-active %}active{% endblock %}
-{% block project_edit_content_header %}{% trans "Notifications" %}{% endblock %}
+{% block project_edit_content_header %}{% trans "Email Notifications" %}{% endblock %}
 
 {% block project_edit_content %}
   <p class="help_text">
-    {% trans "Configure Notifications to be sent on build failures." %}
+    {% trans "Configure email notifications to be sent on build failures." %}
   </p>
 
+  {% if has_old_webhooks %}
+  <p class="help_text">
```

Comment:
These could have a "warning" class or something more prominent, but don't think we are using a class like that in somewhere else.